AnkiDroid/src/main/java/com/ichi2/anki/Info.java

```diff
@@ -177,9 +177,17 @@ private void finishWithAnimation() {
      * @return debugInfo
      */
     public String copyDebugInfo() {
+        String schedName = "Not found";
+        try {
+            schedName = getCol().getSched().getName();
+        } catch (Exception e) {
```

Comment:
I second the motion mentioned by @david-allison-1 for complete catch here, this needs to never fail as we are already in a situation (copying debug info to ask for help) where almost anything could be going wrong.

```suggestion
        } catch (Throwable e) {
```

Very rare to catch Throwable, but here seems justified?

Reply:
I didn't even know Throwable exists

Reply:
It is generally not recommended to catch and/or attempt to handle Throwable as it is the union of Exception (you might be able to recover...) and Error (you are probably dead and need to let it bubble out and crash quickly). It may not even be the right thing to do in this case, it's hard to say - as in it might just result in crashes one line away. But the risk seems low and since we're already sort of in an error-handling case, maybe useful.